packages/cli/src/commands/login.ts

```diff
@@ -68,8 +68,8 @@ export class Authenticator {
                 '/api/oauth/random',
             );
             this.randomChallenge = randomCodeChallenge.data;
-        } catch {
-            throw new Error('An error occured while getting the random challenge.');
+        } catch (error) {
+            throw new Error(`An error occured while getting the random challenge: ${(error as Error).message}`);
```

Comment:
`error` might not be `Error`, surround the line with `if` to make sure (and then we can remove the `as Error`)